project/infra/code-index/src/cli.py

```
@@ -0,0 +1,260 @@
+
+    try:
+        result = db._execute_command("DELETE FROM CodeChunk")
+        count = result.get("count", 0)
+        console.print(f"[green]Deleted {count} chunks[/green]")
+    except Exception as e:
+        console.print(f"[red]Error: {e}[/red]")
+
+
+@app.command()
+def serve(
+    host: str = typer.Option("0.0.0.0", "--host", "-h", help="Host to bind"),
```

Ruff (0.15.2)

[error] Possible binding to all interfaces

(S104)